iXBRLViewerPlugin/viewer/src/js/factset.js

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export function FactSet(facts) {
+    this._facts = facts;
+}
+
+FactSet.prototype._allDimensions = function() {
```

Comment:
Suggest `/* Returns the dimension names relevant to some fact in the set. */`?

Reply:
Improved in d617a3f